### ISSUE TITLE
Como divulgar bem uma vaga?

### ISSUE THREAD

**Issue by klauskpm** [training-center/forum#81]
### 🚀 Objetivo
Usar o feedback da comunidade para ajudar o RH a criar vagas mais interessantes.

### 📔 Descrição
Eu tenho ajudado na divulgação de algumas vagas, mas, vejo muita gente reclamando que o retorno não é o esperado, e muitas vezes não tem retorno. Do outro lado, vejo também desenvolvedoras e desenvolvedores que postam jobs absurdos como piadas, e sei que tem coisas que fazem você ignorar a vaga no piscar de olhos. As vezes nem é a própria vaga, mas, algo que vocês veem da empresa que não agrada.

----

Então, vamos compartilhar o que gostamos de ver em vagas, coisas que nos fazem revirar os olhos e ignorar a vaga (desenvolvedor ninja 🙄), e até vagas que achamos ser puro amorzinho ❤️.

De quebra, se você consegue ler em inglês, esse artigo tem ótimos pontos sobre criação de vagas para desenvolvedores: https://www.codementor.io/blog/software-engineer-job-descriptions-that-attract-the-best-developers-241lev4cs8

> **Pontos listados no artigo:**
> - Título da vaga, local, tipo de contratação (CLT, PJ) e variação salarial
> - Uma visão geral da visão da compania, missão e cultura
> - Uma lista detalhada com as responsabilidades do dia-a-dia
> - Qualificações requiridas e boas a ter/diferênciais
> - Benefícios

----

#### Seja inclusivo
[Use títulos e descrições que abranjam todos](https://github.com/training-center/forum/issues/81#issuecomment-411144699). [O mesmo vale para as imagens](https://github.com/training-center/forum/issues/81#issuecomment-411147087). Se for usar imagem de uma pessoa, [tenha diversidade](https://github.com/training-center/forum/issues/81#issuecomment-411147642). Veja o [relato da ThoughtWorks sobre linguagem neutra de de gênero](https://github.com/training-center/forum/issues/81#issuecomment-411163375).

#### Explique bem a responsabilidade e inclua sua stack tecnológica
[Não precisar ser uma descrição longa, e muito menos pedir que a pessoa saiba tudo](https://github.com/training-center/forum/issues/81#issuecomment-411141523). Você pode usar várias tecnologias, e é ótimo encontrar alguém que saiba todas elas, mas, [foque nas suas principais e deixe claro o que é requirido e o que é um adicional](https://github.com/training-center/forum/issues/81#issuecomment-411158371). Além disso, [procure pessoas com o conhecimento de negócio/funcionamento, em vez de ferramentas](https://github.com/training-center/forum/issues/81#issuecomment-411161884).

#### Exiba o salário, ou a variação dele pra vaga
[Pessoas que moram longe precisam saber se vale a pena ir a entrevista, ou fazer a mudança para um local mais próximo](https://github.com/training-center/forum/issues/81#issuecomment-411174903). [A empresa já sabe o quanto vai pagar](https://github.com/training-center/forum/issues/81#issuecomment-411181004), ou [pode ter uma idéia de até onde pode negociar](https://github.com/training-center/forum/issues/81#issuecomment-411196190). A única coisa que você vai acontecer, é que a empresa vai perder dinheiro fazendo entrevistas que são nãos garantidos.

#### Fala sobre a empresa, sua visão e cultura
[As pessoas podem não se alinhar a sua cultura](https://github.com/training-center/forum/issues/81#issuecomment-411222794), e isso pode, de novo, só resultar em perdas pros dois lados. Gastando tempo em entrevista, insatisfação de ambos os lados e tempo curto de contratação. Então, [se inspire em pessoas que fazem bem o trabalho de criar uma boa descrição da sua cultura](https://github.com/training-center/forum/issues/81#issuecomment-412073999).

#### Fale sobre seus benefícios
[Explique se você tem home office, ou se é negociável](https://github.com/training-center/forum/issues/81#issuecomment-411626408), se tem horários flexíveis, plano de saúde, ajuda com estudos e muitos outros pontos. Atraia pessoas com benefícios que façam a diferença.

#### Evite criar vagas confidênciais
[Elas assustam](https://github.com/training-center/forum/issues/81#issuecomment-411222794)! Não passam confiança porque nós não sabemos os motivos de ser confidêncial. A pessoa vai descobrir qual é o nome da empresa na entrevista. [Veja mais sobre](https://github.com/training-center/forum/issues/81#issuecomment-411251430).

#### Outros posts destacados
Siga todas as dicas [desse comentário](https://github.com/training-center/forum/issues/81#issuecomment-411180790) para ter uma vaga, e processo, com ótimos padrãoes, além de saber como evitar que sua vaga, e empresa, virem piada pelo absurdo.
Pode usar como base, para depois melhorar, [esse comentário](https://github.com/training-center/forum/issues/81#issuecomment-411614764) com o exemplo de uma divulgação de vaga com o feedback coletado aqui.

**Comment by ruanvalente** [comment]
On Aug 7, 2018 14:13, "Klaus Kazlauskas" <notifications@github.com> wrote:

🚀 Objetivo

Usar o feedback da comunidade para ajudar o RH a criar vagas mais
interessantes.
📔 Descrição

Eu tenho ajudado na divulgação de algumas vagas, mas, vejo muita gente
reclamando que o retorno não é o esperado, e muitas vezes não tem retorno.
Do outro lado, vejo também desenvolvedores que postam jobs absurdos como
piadas, e sei que tem coisas que fazem você ignorar a vaga no piscar de
olhos. As vezes nem é a própria vaga, mas, algo que vocês veem da empresa
que não agrada.
------------------------------

Então, vamos compartilhar o que gostamos de ver em vagas, coisas que nos
fazem revirar os olhos e ignorar a vaga (desenvolvedor ninja 🙄), e até
vagas que achamos ser puro amorzinho ❤️.

De quebra, se você consegue ler em inglês, esse artigo tem ótimos pontos
sobre criação de vagas para desenvolvedores:
https://www.codementor.io/blog/software-engineer-job-descriptions-that-attract-the-best-developers-241lev4cs8

*Pontos listados no artigo:*

   - Título da vaga, local, tipo de contratação (CLT, PJ) e variação
   salarial
   - Uma visão geral da visão da compania, missão e cultura
   - Uma lista detalhada com as responsabilidades do dia-a-dia
   - Qualificações requiridas e boas a ter/diferênciais
   - Benefícios

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/training-center/forum/issues/81>, or mute the thread

**Comment by edgarberlinck** [comment]
É muito difícil ter uma descrição perfeita e que funcione com qualquer um, mas pessoalmente gosto de ver o seguinte:

- **Uma descrição da vaga que está sendo ofertada, com as respectivas responsabilidades:** Não precisa ser algo longo, mas acho legal quando a empresa é honesta e abre mão da superficialidade, ex: Não precisa dizer que você quer um funcionário dedicado, isso é um pre-requisito para e trabalhar em qualquer lugar. No lugar dessas coisas diga, com honestidade, o tipo de projeto e ambiente que o profissional deverá atuar, isso ai já é um excelente filtro.

- **Mostre qual é a Stack Atual da empresa (ou projeto, se for o caso)**: Eu odeio empresas que colocam praticamente tudo que existe no mercado, não tem sentido algum pedir por um cara que saiba React, Vue, Angular, Ember, Aurélia.... Vagas assim me mostram que a empresa é de fundo de quintal ou um inferno de louca. Esse tipo de coisa é sempre legal alinhar com o **corpo técnico da empresa** e não com empresas de recrutamento.

Claro que é sempre legal a empresa ter um site mostrando o seu ambiente, já vi empresas que tinham até videos com os funcionários trabalhando. 

Enfim, caso eu tenha esquecido de alguma coisa posso ir falando no decorrer dessa thread.

**Comment by alinebastos** [comment]
Título e descrição INCLUSIVOS.
Muita gente acha besteira - normalmente homens - mas PRECISAMOS incluir as pessoas TODAS.
Chega palavras predominantemente masculinas!
Pode ser "pessoa desenvolvedora", pode ser "desenvolvedoras e desenvolvedores", o que a imaginação mandar, mas, por favor, parem de achar que usar palavras no masculino vai incluir todas as pessoas.

:)

**Comment by silviarcr** [comment]
Complementando  @alinebastos, pode parecer besteira mas as imagens usadas nos anúncios também são nada inclusivas. Cai sempre nos estereótipo homens brancos, de óculos e magros. Já notaram isso?

**Comment by juuh42dias** [comment]
E fazendo um adendo @silviarcr  estilo hipster lenhador também.

**Comment by klauskpm** [comment]
Concordo plenamente @edgarberlinck. Eu só acho justo quando falam de várias tecnologias quando falam de um conhecimento diferencial, como conhecer uma ou mais tecnologias, ou quando falam sobre a pessoa conhecer pelo menos uma das tecnologias.

Perfeito @alinebastos! Aproveitei pra alterar o texto e ser inclusivo também 😬

**Comment by edgarberlinck** [comment]
Tem outra coisa também que me incomoda: **Humor forçado**. Uma vez apliquei para uma empresa e ela fazia perguntas sobre star wars.... Cara, eu odeio star wars..... A não ser que o cargo seja para desenvolver algo **direta ou indiretamente** ligado ao star wars não vejo sentido nisso.

Acho legal ser uma empresa descontraída, mas acho que essas coisas podem não surtir o efeito desejado.

**Comment by silviarcr** [comment]
Achei interessante falar sobre Stack. Pedem x,y,w,z,k e contratam quem se arriscou, mandou CV e sabe x e um pouquinho de y

**Comment by mauricioabreu** [comment]
Quando falar sobre ferramentas, não fale sobre ferramentas. haha

Eu vejo muita vaga falando que "precisa conhecer RabbitMQ".

Bom, quem conhece o conceito de filas, de mensageria, de exchange, de ack, de HTTP, polling, vai manjar como o RabbitMQ funciona como FERRAMENTA.

Eu aplico isso pra todas as ferramentas. Linguagens são mais complexas de abstrair dessa maneira, mas é lamentável pedir conhecimento em ferramentas.

**Comment by woliveiras** [comment]
A TW tem um post maneiro sobre linguagem neutra para vagas: [Adotando a linguagem neutra de gênero](https://medium.com/coragem/adotando-a-linguagem-neutra-de-g%C3%AAnero-e509e6e4e06c)

**Comment by locemarn** [comment]
Colocar salário e não "à combinar".
Isso me deixa fullPistola!!rss
Como se aplicar a uma vaga sem saber se realmente vale a pena financeiramente??
Há pessoas de cidades próximas a vaga oferecida que precisam saber se vale uma mudança de cidade ou se vai dar p/ pagar o fretado.......
Creio que empresas e devs deixariam de perder tempo com essa """simples""" informação. 
:)

**Comment by klauskpm** [comment]
Valeu @woliveiras. O texto é muito bom mesmo. Repassei aqui 😬

~Eu só me preocupo com a pessoa achar a vaga quando é usada a linguagem neutra de gênero. Pesquisando por _desenvolvedora_, eu consigo achar resultados para: desenvolvedora, pessoa desenvolvedora e desenvolvedor. Procurando por _desenvolvedor_, eu cheguei na quinta página sem ver um "desenvolvedora" - sim, triste. Tenho medo que isso faça a pessoa perder uma vaga que seria ótima, por pensar na inclusão dela, por não estar habituada a procurar desse jeito.~

~Eu fui apresentado agora ao conceito de linguagem neutra de gênero para divulgação de vagas, então, vou procurar mais fontes para argumentar a favor \o/~

**Atualização:**
Percebi que a lógica que usei na verdade não é válida, e que devemos sim ser mais inclusivos justamente para poder mudar a forma como procuramos esses empregos. Foi mal por não ter percebido o óbvio o/

**Comment by munizart** [comment]
### Coisas pra se ter em mente ao divulgar uma vaga e tocar o processo seletivo (Na minha opinião)
 - Nem todo mundo vive numa realidade que pode parar o que ta fazendo pra responder seu desafio em 24hrs ou em um final de semana
 - Deixar claro qual o nível da vaga
 - Deixar, no mínimo, uma faixa de remuneração (e caso for uma bolsa estagio, se ela existe)
 - Deixar claro qual o conhecimento principal procurado e quais podem ser desenvolvidos dentro da empresa
 - Nunca criar vagas como "Estagio fullstack"
 - Bom ambiente de trabalho e ambiente inclusivo não são benefícios, é o mínimo esperado.
 - Ser sincero.
 - Dar feedback
 - Ler o [vagas arrombadas](https://twitter.com/vagasvtnc) e reler sua vaga depois pra ver se bate no desconfiômetro

**Comment by silviarcr** [comment]
Muito bem observada a questão do salário. A empresa sabe quanto quer pagar. Aí fica nessa de negociar e acaba em pessoas na mesma posição com salários diferentes. Nem vou entrar no debate sobre salários de mulheres serem mais baixos do que de homens (opa, já entrei no debate. Rs)

**Comment by edgarberlinck** [comment]
Sobre a questão das vagas com descrição neutra eu estou com @klauskpm. Sobre a questão do salário eu discordo: Você que tem que saber o quanto que você vale e também estar disposto a conversar com a empresa.

Até pq a empresa pode oferecer apenas R$ 3000 de salário, entrevistar uma pessoa que quer R$ 5000, gostar e resolver pagar (mesmo não estando em seu orçamento). Pode ser que a empresa entre em contato depois de um tempo (Quando for possível pagar o que a pessoa pediu). Pode ser que a empresa e a pessoa entrem num acordo.

Acho que é assim: Você está buscando uma oportunidade você tem que se prestar a entrevista para conhecer a empresa e deixar a empresa conhecer vc. Salário acaba virando uma barreira que impede essa relação.

**Comment by mauricioabreu** [comment]
Concordo @edgarberlinck 

A empresa que se preze sempre tem um mínimo e um máximo pra pagar. Se a pessoa sabe e vai produzir mais, essa pessoa pode pedir mais dinheiro. Aqui na empresa tem um mínimo pra cada vaga que está aberta. Mas, dependendo da pessoa, pode ser mais.

**Comment by juuh42dias** [comment]
Adicionando mais uma, vagas confidenciais me assustam, eu só passo reto e nem leio.

SE possível a empresa poderia deixar mais claro quanto a sua cultura dentro da empresa e ou que se espera do(a) profissional que está contratando. Vejo e já vivi casos em que eu literalmente não tinha o perfil da empresa, porém só fomos descobrir isso quando trabalhando. Entrevistas de fit podem ser chatas, mas ajudam no processo seletivo. Se você já ñ explicitar isso na vaga, pode ser um problema durante o processo ou depois dele.

**Comment by klauskpm** [comment]
Concordo com as vagas confidenciais @juuh42dias. Sempre passei longe delas. Nem olhava a descrição.

Sobre a cultura, sempre acho bom saber sobre:
- Ter voz para dar prazos e decisões técnicas
- Qualidade de código (PR, lint, CI/CD, Pair programming, testes)
- Horas extras
- Divisão de times/tribos

Às vezes a pessoa saí com um mês de trabalho por causa dessa falta de alinhamento. Péssimo pra pessoa e pra empresa. Ninguém ganha no final.

**Comment by AndersonMarquess** [comment]
Não é difícil encontrar esse tipo de vaga como citado pela @juuh42dias , chega a ser bizarro:
**Empresa**: confidencial.
**Endereço**: confidencial.
**Salário**: a combinar.

PS: tem um ep do _sudo cast_ sobre recrutamento na área de tecnologia: http://sudocast.com.br/portfolio-items/ep-0009-techrecruiting/

**Comment by klauskpm** [comment]
Atualizando [meu post anterior](https://github.com/training-center/forum/issues/81#issuecomment-411178962)...

Deve ser feita sim a inclusão. Deve ser feito sim o uso de **pessoa programadora**. Justamente para as pessoas mudarem a forma de buscar e ter vagas mais coerentes.

**Comment by klauskpm** [comment]
Tentando seguir o feedback daqui, criei a seguinte vaga:
```
Pessoa desenvolvedora front-end sênior - CLT
Bairro - Rio de Janeiro, RJ
Salário entre R$ X.000,00 e R$ Y.000,00

Visão/Cultura:
----

Responsabilidades:
- Desenvolver e dar manutenção a sistemas de vendas
- Desenvolver e dar manutenção a pacotes de NPM da equipe e da empresa
- Alinhar estruturas de layout com a equipe de design
- Fazer testes unitários e de integração

Tecnologias que usamos:
- JavaScript (ES6, Node)
- Angular 6, React, Sass, Jasmine, Jest
- Webpack, Gulp

Qualidades necessárias:
- Domínimo de versionamento com Git
- Excelente conhecimento em HTML, CSS (SASS/LESS) e JavaScript
- Grande conhecimento em Angular ou React

Qualidades extras:
- Conhecimento em desenvolvimento de testes com Jasmine ou Jest
- Automatização de tarefas e bundling, preferencialmente com Webpack

Benefícios:
- Plano de saúde X
- Plano dental Y
- Vale transporte
- Desconto de 40% na faculdade X
- Horário flexível
- Banco de horas
- Disponibilidade de combinar home office
- Palestras internas (Desenvolvimento, Agile, Workshops)
- Reuniões de melhoria e levantamento de novas tecnologias
```

O que pode melhorar aqui?

Eu deixei a visão e cultura em branco porque não consegui pensar no que escrever. Alguém teria um bom exemplo pra dar? 😬

**Comment by viniciuslagedo** [comment]
Um ponto que também acho importante hoje em dia é deixar claro a possibilidade de home office ou não. E também do horário de trabalho. Recentemente uma amiga fez um processo e só soube depois que estava contratada que tinha escalas nos finais de semana.

EDIT: Só vi teu último post agora @klauskpm ignora a parte do home office XD

**Comment by klauskpm** [comment]
Ignoro não 😛. É importantíssimo mesmo. Às vezes a pessoa tem condições especiais, como precisar cuidar de um parente, e perdem um bom tempo filtrando vagas que tem, ou que negociam, mas não declaram.

**Comment by lgssouza** [comment]
Escrever sobre a cultura da empresa é algo inovador para as novas vagas, gostei muito do modelo de publicação proposta, no setor de desenvolvimento de software a cultura é diferente, o empregador que busca o colaborador e não o inverso, como trata-se de um processo criativo temos que convencer as pessoas a participar do nosso time, como isso é possível?  Só benefícios financeiros já não são mais suficientes, a nova geração preocupa muito mais com o todo do que com o individual. 
Projetos que fazem a diferença na vida das pessoas são os mais atrativos, eu acredito que a empresa tem que estar disposta a contar sobre seus projetos, sobre a organização em si, forma de trabalho, equipe, apoio a estudos, qualidade de vida em troca da qualidade financeira.

"Quer satisfação maior do que saber que você está quebrando paradigmas e construindo o futuro de um país? Na Softplan seu trabalho não é mais um. Aqui você o vê fazer a diferença e impactar no dia a dia de milhares de brasileiros que utilizam nossos sistemas – até mesmo sem saber."
Trecho retirado da empresa Softplan: https://www.softplan.com.br/carreira/

Acho que é uma ideia válida de cultura.

**Comment by klauskpm** [comment]
Ótimo comentário @lgssouza.

Aproveitei para gerar um compilado, no post original, de todos os pontos mais importantes citados aqui.
